trackpoint/src/module.c

```diff
@@ -21,22 +21,19 @@ void Module_Init(void)
                       &(port_pin_config_t){/*.pullSelect=kPORT_PullDown,*/ .mux=kPORT_MuxAsGpio});
     GPIO_PinInit(PS2_CLOCK_GPIO, PS2_CLOCK_PIN, &(gpio_pin_config_t){.pinDirection=kGPIO_DigitalInput, .outputLogic=0});
 
-    PORT_SetPinInterruptConfig(PS2_CLOCK_PORT, PS2_CLOCK_PIN, kPORT_InterruptEitherEdge);
+    PORT_SetPinInterruptConfig(PS2_CLOCK_PORT, PS2_CLOCK_PIN, kPORT_InterruptFallingEdge);
```

Comment:
All following changes (all in the `module.c`) are not so important:

- They are great deal aesthetic changes.
- The PS2 interrupt will now be called only on falling edge. This means lower probability of timing issues as there is fewer calls overall. (All related changes should be aesthetic only.)
- It should now be more robust and be able to deal even with severe problems with either timing or garbled GPIO read values.

During testing, it seems that some evidence of timing-induced race conditions was encountered - i.e., some PS2 interrupts arriving late - but the evidence was not very conclusive and fixing the other race condition (the one on 16-bit assignments) seems to have fixed all the problems in practice, so while changes in `module.c` might have some effect on overall robustness, they are by no means mandatory part of the PR. (I.e., feel free to pick whichever set of the changes seems reasonable.)